dev-libs/tinyxml/tinyxml-2.6.2.recipe

```diff
@@ -57,7 +57,24 @@ INSTALL()
 	cp -R tiny*.h $includeDir
 	cp libtinyxml.so* $libDir
 
-	prepareInstalledDevelLibs libtinyxml
+	prepareInstalledDevelLib libtinyxml
+
+	# add missing pkgconfig file
+	mkdir -p $developLibDir/pkgconfig
+	cat > $developLibDir/pkgconfig/tinyxml.pc << EOF
+prefix=$prefix
+exec_prefix=$prefix
+libdir=$libDir
+includedir=$includeDir
+
+Name: TinyXML
+Description: A simple, small, C++ XML parser
+Version: $portVersion
+Libs: -L\${libdir} -ltinyxml
+Cflags: -I\${includedir}
+EOF
+
+	fixPkgconfig
```

Comment:
Why is `fixPkgconfig` needed if you are writing it directly to $developLibDir?

Reply:
Left-over from previous attempts I think ;) Will remove (after some testing)